script/delete_weird_company.rb

```diff
@@ -0,0 +1,31 @@
+require File.expand_path('../../config/environment', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.